tidb-troubleshooting-map.md

```diff
@@ -582,12 +582,10 @@ Check the specific cause for busy by viewing the monitor **Grafana** -> **TiKV**
 
 - 7.1.5 `distsql.go` reports `inconsistent index`.
 
-    The data index seems to be inconsistent. Run the `admin check table <TableName>` command on the table where the reported index is. If the check fails, close GC by running the following command, and [report a bug](https://github.com/pingcap/tidb/issues/new?labels=type%2Fbug&template=bug-report.md):
+    The data index seems to be inconsistent. Run the `admin check table <TableName>` command on the table where the reported index is. If the check fails, disable garbage collection by running the following command, and [report a bug](https://github.com/pingcap/tidb/issues/new?labels=type%2Fbug&template=bug-report.md):
 
     ```sql
-    begin;
-    update mysql.tidb set variable_value='72h' where variable_name='tikv_gc_life_time';
-    commit;
+    SET GLOBAL tidb_gc_enable = 0;
```

Comment:
Currently this works for the recover table feature. It's ok to use it here, but I don't recommend it.
By the way, actually I think the recover table feature can use service safepoint to prevent GC instead of this. Then this can be safely used as a switch of GC.

Reply:
This is for a table corruption issue, so I changed the example, since setting the lifetime to 72h does not completely handle the case if the user is slow or gets distracted. I think it should be a "stop the world".